### ISSUE TITLE
Pypi update

### ISSUE THREAD

**Issue by simons-public** [trainman419/python-cec#32]
Is there any chance you could do a version bump and upload to pypi? The current version on Pypi doesn't have commit fb230ae4b951dd156616c3b113c080f52b43d057, so it won't compile when installing with pip.

Thanks in advance

**Comment by nforro** [comment]
I've released version 0.2.7, however I don't have access to PyPI. @trainman419 could you upload it?

@simons-public in the meantime you can do:
```
pip install git+https://github.com/trainman419/python-cec.git@0.2.7#egg=cec
```

**Comment by trainman419** [comment]
Sorry I've been slow on this. I've pushed version 0.2.7 to PyPi: https://pypi.org/project/cec/0.2.7/
